crates/forge/tests/e2e/running.rs

```diff
@@ -942,3 +944,64 @@ fn available_gas_error() {
         "}
     );
 }
+
+#[test]
+fn invcompatible_snforge_std_version_warning() {
+    let temp = setup_package("simple_package");
+    let manifest_path = temp.child("Scarb.toml");
+
+    let mut scarb_toml = fs::read_to_string(&manifest_path)
+        .unwrap()
+        .parse::<Document>()
+        .unwrap();
+    scarb_toml["dependencies"]["snforge_std"]["path"] = Item::None;
```

Comment:
Is it needed?

Reply:
yest, otherwise git/path will be ambiguous and scarb will rise an error

Reply:
Forgot we have the path there by default